lib/bronto/contact.rb

```diff
@@ -20,11 +20,11 @@ def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, includ
       Array.wrap(resp[:return]).map { |hash| new(hash) }
     end
 
-    def self.save(*objs)
+    def self.save(endpoint, *objs)
```

Comment:
Yes - This was initially what prompted me to inquire about condition and reliability of the aged tests and if we wanted to bring them up to date, since they are all about 4 years old. Base and Contact both have save methods (contact overrides Base's). 

There do not seem to be other instances where contact's save gets called however. 

There are multiple calls to @contact.save in the tests though so if we are reliant upon them for any sense of security it may makes sense get them working again.